prov/efa/src/rdm/efa_rdm_ope.c

```diff
@@ -1088,7 +1088,8 @@ void efa_rdm_ope_handle_recv_completed(struct efa_rdm_ope *ope)
 			efa_rdm_rxe_handle_error(rxe, -err, FI_EFA_ERR_PKT_POST);
 			efa_rdm_rxe_release(rxe);
 		}
-
+		if (rxe->internal_flags & EFA_RDM_OPE_READ_NACK)
```

Comment:
Why not just move the `efa_rdm_rxe_map_remove` in L1110 before the DC flag check?

Reply:
I wasn't sure if it is safe. I guess it is fine since rxe is not touched.